tunnel/tun.sh

```diff
@@ -17,5 +17,7 @@ open_ssh_tunnel () {
 
 }
 
+chmod 600 ${DOCKER_CONTAINER_SSH_KEYS}
```

Comment:
add a comment as to why we're doing this

Reply:
#52 adds an explanation